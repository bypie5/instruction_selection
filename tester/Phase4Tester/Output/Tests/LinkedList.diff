--- SelfTestCases/LinkedList.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/LinkedList.out	2020-03-14 21:20:45.000000000 -0700
@@ -14,12 +14,15 @@
 39
 25
 2220000
--555
--555
+555
+555
 28
 22
 25
 33300000
+555
+555
+28
 22
 25
 44440000
