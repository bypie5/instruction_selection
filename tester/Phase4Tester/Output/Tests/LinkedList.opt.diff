--- SelfTestCases/LinkedList.opt.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/LinkedList.opt.out	2020-03-14 18:57:59.000000000 -0700
@@ -1,27 +1,6 @@
-25
-10000000
-39
-25
-10000000
-22
-39
-25
-1
-0
-10000000
-28
-22
-39
-25
-2220000
--555
--555
-28
-22
-25
-33300000
-22
-25
-44440000
-0
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 16 column 1: "li": Too many or incorrectly formatted operands. Expected: li $t1,-100
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 297 column 10: "1": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 331 column 10: "555": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 348 column 10: "555": operand is of incorrect type
 
+Processing terminated due to errors.
