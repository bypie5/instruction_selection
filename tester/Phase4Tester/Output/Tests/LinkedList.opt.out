Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 16 column 1: "li": Too many or incorrectly formatted operands. Expected: li $t1,-100
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 297 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 331 column 10: "555": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/LinkedList.opt.compiler.out line 348 column 10: "555": operand is of incorrect type

Processing terminated due to errors.
